### PR TITLE
FEATURE: BB-643 Add headlines secondary measure

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/884/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/724/ : SUCCESS in 52s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/195/ : SUCCESS in 26s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/231/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/885/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/235/ : UNSTABLE in 5m 26s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/232/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/886/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/236/ : UNSTABLE in 3m 25s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/887/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/725/ : SUCCESS in 51s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/196/ : SUCCESS in 33s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/233/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/888/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/237/ : UNSTABLE in 3m 26s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/234/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/889/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/238/ : SUCCESS in 3m 10s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1001/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/806/ : FAILURE in 1m 57s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/279/ : SUCCESS in 3m 03s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/282/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1002/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/282/ : UNSTABLE in 6m 36s

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "LICENSE"
   ],
   "devDependencies": {
-    "@gooddata/mock-js": "1.0.0-alpha3",
+    "@gooddata/mock-js": "v1.0.0-alpha4",
```
Comment (martinmosko) [inline package.json]: "v" ?

File: src/execution/fixtures/ExecuteAfm.fixtures.ts [hunk 2]
```diff
@@ -425,6 +425,59 @@ const oneMeasureResponse: Execution.IExecutionResponses = {
...
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ],
+            links: {
+                // tslint:disable-next-line:max-line-length
+                executionResult: '/gdc/app/projects/d20eyb3wfs0xe5l0lfscdnrnyhq1t42q/executionResults/2651138797087227392'
```
Comment (martinmosko) [inline src/execution/fixtures/ExecuteAfm.fixtures.ts]: Its not necessary to use real link. Then you can get rid of tslint comment as well

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/289/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1021/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/287/ : UNSTABLE in 5m 21s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1029/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/827/ : SUCCESS in 2m 34s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/300/ : SUCCESS in 2m 47s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/291/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1030/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/290/ : UNSTABLE in 5m 52s

**Comment by petrjanu** [comment]
extended test - cafe

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1032/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/291/ : FAILURE in 14s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1031/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/828/ : SUCCESS in 2m 28s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/301/ : SUCCESS in 3m 04s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/292/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1033/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/292/ : SUCCESS in 4m 04s

**Pull request closed without merging**